Quantum-Control-Applications/ESR/cpmg.py

```diff
@@ -0,0 +1,206 @@
+from qm import SimulationConfig
+from qm.qua import *
+from qm import LoopbackInterface
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+import matplotlib.pyplot as plt
+from qm.simulate.credentials import create_credentials
+
+################################
+# Open quantum machine manager #
+################################
+
+qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
+
+########################
+# Open quantum machine #
+########################
+
+qm = qmm.open_qm(config)
+
+###################
+# The QUA program #
+###################
+
+tau_min = 2000//4  # tau is half the time between pi pulses
+tau_max = 5e4//4  # tau is half the time between pi pulses
+tau_array = np.round(np.exp(np.linspace(np.log(tau_min), np.log(tau_max), 30)))  # outputs results in float
```

Comment:
I played with it and keep running into the same issue that I need to add np.log() and change the base to np.exp(1). np.logspace starts at (base**tau_min) not at tau_min, if it were to start at tau_min and end at tau_max then do logspacing it would be fantastic

Reply:
You could write like this: `np.round(np.logspace(np.log10(tau_min), np.log10(tau_max),30))` which gives you the exact same results

Reply:
thanks